doc/march_packages/march_gain_scheduling.rst

```diff
@@ -0,0 +1,65 @@
+.. _march-march_gain_scheduling-label:
+
+march_gain_scheduling
+=====================
+
+The march_gain_scheduling package is used to implement the gainscheduling control method. More about this control method
+can be found on `this confluence page <https://confluence.projectmarch.nl:8443/display/51/31+-+Gain+Scheduling>`_.
```

Comment:
This documentation is public, therefore, it would be nice that every URL you link to is also public. Otherwise people would think they miss vital information. So, either move the gain scheduling documentation here or remove it.

Reply:
I will remove the link